x-pack/plugins/endpoint/server/routes/alerts/list/lib.ts

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { get } from 'lodash';
+import { encode, decode, RisonValue } from 'rison-node';
+import { SearchResponse } from 'elasticsearch';
+import { KibanaRequest } from 'kibana/server';
+import { Direction } from '../../../../../../legacy/plugins/siem/public/graphql/types';
```

Comment:
I don't think it's safe to import types from other kibana plugins. We should define our own if there isn't a Kibana core platform type for this.

Reply:
@peluja1012 Isn't that the point of public? To indicate that things are safe for export? I could be wrong... In any case, I'm happy to redefine this locally.